pkg/cli/version/version.go

```diff
@@ -86,8 +86,12 @@ func NewCmdVersion(f cmdutil.Factory, ioStreams genericclioptions.IOStreams) *co
 
 // Complete is copied from upstream version command with added clusteroperator client
 // to report OpenShift server version
-func (o *VersionOptions) Complete(f cmdutil.Factory, cmd *cobra.Command) error {
-	var err error
+func (o *VersionOptions) Complete(f cmdutil.Factory, cmd *cobra.Command, args []string) error {
+	err := o.Options.Complete(f, cmd, args)
```

Comment:
This is a post-rebase fix. The upstream `Validate(...)` doesn't accept `args` anymore. Another solution is to copy the upstream implementation of `Validate(...)` as we did with `Complete(...)`.